glide.yaml

```diff
@@ -11,7 +11,7 @@ import:
 - package: github.com/gorilla/mux
   version: ~1.1.0
 - package: github.com/prometheus/client_golang
-  version: ~0.8.0
+  version: master
```

Comment:
Master for our stuff makes sense to me, not for external deps. I have problems with glide quite often because the cobra/pflag deps are shite and used everywhere, and everywhere differently.